.github/workflows/build.yml

```diff
@@ -143,3 +143,24 @@ jobs:
       - uses: actions/upload-artifact@v2
         with:
           path: ./wheelhouse/*.whl
+  wheel-arm64:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["macOS-latest"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install deps
+        run: python -m pip install -U cibuildwheel==2.1.2
+      - name: Build Wheels
+        env:
+          CIBW_SKIP: "cp310-* pp*"
```

Comment:
This line will need updating in #1430 after merge, and similar in `deploy.yml`.

Reply:
Yep, I'll update it here or in #1430 depending on which merges first

Reply:
Actually on second thought it's not strictly needed as the cibuildwheel configuration already filters arm64 macOS wheels to >= 3.8 as those are the only python versions which officialy support arm64 macOS. So we don't strictly need to change anything here because it'll already skip 3.6 and 3.7